src/routes/submit/submit.js

```diff
@@ -47,13 +46,16 @@ export const submit = [
   }
 ]
 
-const getHandler = () => {
+const getHandler = async (application) => {
```

Comment:
why is it async now?

Reply:
will change - was because of a dynamic import